designer/server/src/main/scala/pl/touk/nussknacker/ui/process/ScenarioActivityAuditLog.scala

```diff
@@ -0,0 +1,221 @@
+package pl.touk.nussknacker.ui.process
+
+import com.typesafe.scalalogging.LazyLogging
+import pl.touk.nussknacker.engine.api.deployment._
+import pl.touk.nussknacker.engine.api.process.{ProcessId, VersionId}
+import pl.touk.nussknacker.ui.process.ScenarioAttachmentService.AttachmentToAdd
+import pl.touk.nussknacker.ui.security.api.LoggedUser
+
+object ScenarioActivityAuditLog extends LazyLogging {
+
+  def onCreateScenarioActivity(
+      scenarioActivity: ScenarioActivity
+  ): Unit =
+    logger.info(
+      withPrefix(scenarioActivity.scenarioId, scenarioActivity.scenarioVersionId, scenarioActivity.user.name.value)(
+        s"New activity: ${printScenarioActivity(scenarioActivity)}"
+      )
+    )
+
+  private def printScenarioActivity(scenarioActivity: ScenarioActivity) = scenarioActivity match {
```

Comment:
1. it'd be good to move the private methods behind the public ones
2. `print` or maybe `stringify`?